resources/views/templates/profile/password/form.blade.php

```diff
@@ -0,0 +1,71 @@
+                        @foreach($columns as $col)@if($col['name'] == 'password')<div class="form-group row align-items-center" :class="{'has-danger': errors.has('{{ $col['name'] }}'), 'has-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid }">
+                            <label for="{{ $col['name'] }}" class="col-form-label text-md-right" :class="isFormLocalized ? 'col-md-4' : 'col-md-3'">{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}</label>
+                            <div :class="isFormLocalized ? 'col-md-4' : 'col-md-9 col-xl-7'">
+                                <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'required|{{ implode('|', collect($col['frontendRules'])->reject(function($rule) use ($col) { return $rule === 'confirmed:'.$col['name'].'_confirmation';})->toArray()) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}" ref="{{ $col['name'] }}">
```

Comment:
This looks like a mistake: $rule === 'confirmed:'.$col['name'].'_confirmation'; Should be $rule === 'confirmed:'.$col['name'];

Reply:
Fixed in next commit.